Quake/Makefile

```diff
@@ -176,7 +176,7 @@ endif
 COMMON_LIBS:= -lm
 
 ifneq ($(HOST_OS),darwin)
-COMMON_LIBS+= -lX11-xcb -lvulkan
+COMMON_LIBS+= -lX11-xcb -lvulkan -lMoltenVK
```

Comment:
Why are you adding -lMoltenVK to the non-darwin case and breaking linux, etc builds?

Reply:
@sezero yeah, *opps* sorry ... it was actually a long time ago I did this. So now that I look back on it, yeah, why did I do that? 🤔

Fixing it in the next commit.